code/game/objects/items/weapons/grenades/plastic.dm

```diff
@@ -130,6 +130,7 @@
 		if(!qdeleted(target))
 			location = get_turf(target)
 			target.overlays -= image_overlay
+			target.priority_overlays -= image_overlay
```

Comment:
I considered it, but that would be unnecessary overhead on top of lots of mostly unnecessary overhead, and also another 72 line changes on a PR that has enough trouble with conflicts already.